app/src/main/java/org/schabi/newpipe/local/holder/LocalBookmarkPlaylistItemHolder.java

```diff
@@ -0,0 +1,54 @@
+package org.schabi.newpipe.local.holder;
+
+import android.view.MotionEvent;
+import android.view.View;
+import android.view.ViewGroup;
+
+import org.schabi.newpipe.R;
+import org.schabi.newpipe.database.LocalItem;
+import org.schabi.newpipe.database.playlist.PlaylistMetadataEntry;
+import org.schabi.newpipe.local.LocalItemBuilder;
+import org.schabi.newpipe.local.history.HistoryRecordManager;
+
+import java.time.format.DateTimeFormatter;
+
+public class LocalBookmarkPlaylistItemHolder extends LocalPlaylistItemHolder {
+    private final View itemHandleView;
+
+    public LocalBookmarkPlaylistItemHolder(final LocalItemBuilder infoItemBuilder,
+                                           final ViewGroup parent) {
+        this(infoItemBuilder, R.layout.list_playlist_bookmark_item, parent);
+    }
+
+    LocalBookmarkPlaylistItemHolder(final LocalItemBuilder infoItemBuilder, final int layoutId,
+                                    final ViewGroup parent) {
+        super(infoItemBuilder, layoutId, parent);
+        itemHandleView = itemView.findViewById(R.id.itemHandle);
+    }
+
+    @Override
+    public void updateFromItem(final LocalItem localItem,
+                               final HistoryRecordManager historyRecordManager,
+                               final DateTimeFormatter dateTimeFormatter) {
+        if (!(localItem instanceof PlaylistMetadataEntry)) {
+            return;
+        }
+        final PlaylistMetadataEntry item = (PlaylistMetadataEntry) localItem;
+
+        itemHandleView.setOnTouchListener(getOnTouchListener(item));
+
+        super.updateFromItem(localItem, historyRecordManager, dateTimeFormatter);
+    }
+
+    private View.OnTouchListener getOnTouchListener(final PlaylistMetadataEntry item) {
+        return (view, motionEvent) -> {
+            view.performClick();
```

Comment:
Why perform click? In other places, too

Reply:
Copied from
https://github.com/TeamNewPipe/NewPipe/blob/a4037a826822a7cbf56619dd9a782e3583d17287/app/src/main/java/org/schabi/newpipe/local/holder/LocalPlaylistStreamItemHolder.java#L127-L137

Reply:
Mmmh, ok, I guess AS gives the `ClickableViewAccessibility` warning otherwise ([never understood what those warnings mean](https://stackoverflow.com/questions/47107105/android-button-has-setontouchlistener-called-on-it-but-does-not-override-perform)).